contracts/interchain-router/src/suite_tests/tests.rs

```diff
@@ -156,4 +168,63 @@ fn test_tick() {
     // assert the expected response attributes and messages
     assert_eq!(expected_messages, resp.messages);
     assert_eq!(expected_attributes, resp.attributes);
+
+    // try to use the fallback method to distribute
+    // explicitly defined denom
+    let err = execute(
+        deps.as_mut(),
+        mock_env.clone(),
+        info.clone(),
+        crate::msg::ExecuteMsg::DistributeFallback {
+            denoms: vec!["usdc".to_string()],
+        },
+    )
+    .unwrap_err();
+
+    assert_eq!(
+        err,
+        NeutronError::Std(cosmwasm_std::StdError::generic_err(
+            "unauthorized denom distribution".to_string()
+        ))
+    );
+
+    // now distribute a valid fallback denom
+    let resp = execute(
+        deps.as_mut(),
+        mock_env,
+        info,
+        crate::msg::ExecuteMsg::DistributeFallback {
+            denoms: vec!["denom1".to_string()],
+        },
+    )
+    .unwrap();
+
+    for msg in resp.messages {
+        assert_eq!(
+            msg,
```

Comment:
I would not use a loop, rather use a specific index in the messages array, and would also check to make sure theres only 1 message.
Because if there will be 2 (by design) you will have to change this rather then update it.